tests/onnxifi/test.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Should be run from Glow source directory.
+# Build Glow in glow/build and build_onnxifi_tests.sh then run this script.
+# Accepts one parameter, the name of the test to run. If no test name is
+# provided then it will run all ONNXIFI tests except those excluded by
+# crashed_test.txt and failed_tests.txt.
+
+GLOW_DIR=$PWD
+GLOW_BUILD_DIR="${GLOW_DIR}/build"
+ONNX_DIR="${GLOW_DIR}/thirdparty/onnx"
+ONNX_BUILD_DIR="${ONNX_DIR}/build_onnx"
+GLOW_ONNXIFI_DIR="${GLOW_BUILD_DIR}/lib/Onnxifi"
+
+# Name of the specific test to run, if empty will run all tests except
+# crashing/failing tests.
+TEST_NAME=$1
+
+# Check that glow/build exists.
+check_glow_build_dir_exist() {
+  if [ ! -d "$GLOW_BUILD_DIR" ]; then
+    echo "Expected Glow to be built in ${GLOW_BUILD_DIR}"
+    exit -1
+  fi
+}
+
+# Copy libonnxifi-glow(.so|.dylib") dynamic library to libonnxifi(.so|.dylib").
+copy_glow_lib() {
+  COPIED=0
+
+  if [ -f "${GLOW_ONNXIFI_DIR}/libonnxifi-glow.dylib" ]; then
+   cp "${GLOW_ONNXIFI_DIR}/libonnxifi-glow.dylib" "${GLOW_ONNXIFI_DIR}/libonnxifi.dylib"
```

Comment:
where do you set ENV VAR for the path to load dll?

Reply:
`LD_LIBRARY_PATH=${GLOW_ONNXIFI_DIR} ...` on line 60

Reply:
@rdzhabarov this works on my mac but I'm not sure why so I'll change it

Reply:
This is likely because `DYLD_LIBRARY_PATH` is set :)

Reply:
I think MacOS also supports LD_LIBRARY_PATH?  But idk why it has DYLD_LIBRARY_PATH too in that case.

Reply:
oh, did not know about that